testutil/utils.go

```diff
@@ -0,0 +1,36 @@
+package testutil
+
+import (
+	"bytes"
+	"context"
+	"encoding/hex"
+
+	"github.com/celestiaorg/celestia-app/pkg/appconsts"
+	"github.com/celestiaorg/nmt/namespace"
+	"github.com/cosmos/cosmos-sdk/client"
+	tmrand "github.com/tendermint/tendermint/libs/rand"
+	rpctypes "github.com/tendermint/tendermint/rpc/core/types"
+)
+
+func RandomValidNamespace() namespace.ID {
+	for {
+		s := tmrand.Bytes(8)
+		if bytes.Compare(s, appconsts.MaxReservedNamespace) > 0 {
+			return s
+		}
+	}
+}
+
+func QueryWithOutProof(clientCtx client.Context, hashHexStr string) (*rpctypes.ResultTx, error) {
+	hash, err := hex.DecodeString(hashHexStr)
+	if err != nil {
+		return nil, err
+	}
+
+	node, err := clientCtx.GetNode()
+	if err != nil {
+		return nil, err
+	}
+
+	return node.Tx(context.Background(), hash, false)
```

Comment:
[Question][Non Blocking]
Why using the background context when we have the `clientCtx`?

Reply:
the clientCtx doesn't actually have a context.Context in it, and I didn't want to force the testnode package as a dependency. This is good point tho